src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/NewEditCyclePage/PayoutsSection/PayoutsTable/modals/DeleteAllPayoutsModal.tsx

```diff
@@ -0,0 +1,34 @@
+import { Trans, t } from '@lingui/macro'
```

Comment:
I have a modal for this somewhere you could probably reuse

Reply:
It's pretty simple to use too! You can wrap something in `handleConfirmationDeletion`, see `src/components/ProjectDashboard/components/Cart/components/CartItem/NftCartItem.tsx:39`

Lemme know if you need any guidance!

Reply:
Nice! I've implemented it and removed all the `DeleteAllPayoutsModal` code.

<img width="1085" alt="Screen Shot 2023-08-08 at 3 59 58 pm" src="https://github.com/jbx-protocol/juice-interface/assets/96150256/243fea23-50f7-47d5-bce2-3ffe407c62af">